motor/core.py

```diff
@@ -85,6 +85,20 @@ def __eq__(self, other):
 
     def __init__(self, delegate):
         self.delegate = delegate
+        warnings.warn(
+            DeprecationWarning(
+                """As of May 14th, 2025,
+        Motor is deprecated in favor of the GA release of the PyMongo Async API.
```

Comment:
Is this warning formatted the way we want when it gets displayed to the user? Could you give an example of what it looks like in the terminal?